### ISSUE TITLE
Port to SDK 1.3

### ISSUE THREAD

**Issue by cmbruns-hhmi** [cmbruns/pyovr#14]
I created a branch sdk_1.3 for upgrading to the recent SDK 1.3 release. I made some progress but at this point I'm getting a runtime error when calling ovr_CreateTextureSwapChainGL(), which is returning error code -1005, which means ovr Error_InvalidParameter.

I'm testing with a DK2. The Oculus Home app seems to get launched automatically during the initial app test run.

**Comment by cmbruns-hhmi** [comment]
Don't forget I've got some manual changes to _ovr130.py, which need to get folded back into translate_headers.pl, once things are working in 1.3.

**Comment by cmbruns-hhmi** [comment]
@jherico I might ask for your help with the sdk_1.3 branch if I can't figure out how to get it working soon. Program **examples/report_hmd_orientation.py** is working with my DK2. But when I try to run, for example, **examples/glut/rift_demo_glut.py**, I am getting return code -1005 (Error_InvalidParameter) when I call **ovr_CreateTextureSwapChainGL**().

**Comment by cmbruns-hhmi** [comment]
I think I found the problem with ovr_CreateTextureSwapChainGL(). I used ovr_GetLastErrorInfo() and got the message "Only texture swap chains of Texture2D type currently supported on PC." This, even though I was passing in ovr.Texture_2D in my ovrTextureSwapChainDesc. It seems my TextureType enum is incorrectly initialized in _ovr130.py. In fact it looks like any enum whose first value is implicitly initialized to zero, it's getting set to +1 instead. This is a serious problem that needs to be fixed in translate_header.pl
